bigquery_datatransfer/nox.py

```diff
@@ -19,8 +19,36 @@
 
 
 @nox.session
-@nox.parametrize('python_version', ['2.7', '3.4', '3.5', '3.6'])
-def unit_tests(session, python_version):
+def default(session):
```

Comment:
@chemelnucfin Can you reduce the copy-paste and make `unit` call `default()`?

Reply:
Fixed this in autogen: googleapis/toolkit#1770.